frontend/src/hooks/usePositioning.js

```diff
@@ -0,0 +1,38 @@
+import {useEffect, useState} from "react";
+
+export default function usePositioning() {
+
+
+    const [geoAvailable, setGeoAvailable] = useState(false);
+    const [latitude, setLatitude] = useState(null);
+    const [longitude, setLongitude] = useState(null);
+    const [errorMessage, setErrorMessage] = useState("");
+
+    const geoSuccess = (position) => {
+        setGeoAvailable(true);
```

Comment:
I think you can get rid of the `geoAvailable` state as it can be derived as `latitide && longitude`, no?